lib/developer_portal/lib/liquid/drops/provider.rb

```diff
@@ -5,7 +5,7 @@ module Drops
     class Provider < Drops::Base
 
       example %(
-        <div>Domain {{ provider.domain }}</div>
+        <div>Domain {{ provider.external_domain }}</div>
```

Comment:
Is this changed intended? :thinking: 
I don't think so, as the method name on line 56 has not changed, and shouldn't be changed, as it will be breaking the interface.

Reply:
Good catch!